server/lib/budget.js

```diff
@@ -588,7 +588,11 @@ export async function sumCollectivesTransactions(
     } else if (transactionType === 'CREDIT_WITH_HOST_FEE_AND_PAYMENT_PROCESSOR_FEE') {
       where = {
         ...where,
-        [Op.or]: [{ type: CREDIT }, { type: DEBIT, kind: 'HOST_FEE' }, { type: DEBIT, kind: 'PAYMENT_PROCESSOR_FEE' }],
+        [Op.or]: [
+          { type: CREDIT },
+          { type: DEBIT, kind: 'HOST_FEE' },
+          { type: DEBIT, kind: 'PAYMENT_PROCESSOR_FEE', OrderId: { [Op.not]: null } },
```

Comment:
Now that I'm looking at it again, would be great to add a comment to explain why we're doing it.